packages/shared/src/lib/config.ts

```diff
@@ -11,3 +11,5 @@ export const fallbackImages = {
   avatar:
     'https://daily-now-res.cloudinary.com/image/upload/f_auto/v1664367305/placeholders/placeholder3',
 };
+
+export const MAX_PRIVILEGED_MEMBERS = 3;
```

Comment:
Perhaps add a prefix like `SQUAD_` or `SQUAD_HEADER_`? Or rename to `MAX_VISIBLE_PRIVILEGED_MEMBERS`, so it can be re-used?

Reply:
I like the last one. Let's go with it 🚀